test/PoolManager.t.sol

```diff
@@ -78,6 +77,16 @@ contract PoolManagerTest is Test, Deployers, GasSnapshot {
         assertEq(address(manager.protocolFeeController()), address(feeController));
     }
 
+    function test_setProtocolFeeController_failsIfNotOwner() public {
```

Comment:
technically i think this will be moved out of this file (like a lot of other tests tbh) but since we haven't migrated the tests out of here I think its fine for this PR. and whoever is assigned with ProtocolFees.t.sol will move it (I think its me so I can do it when I pick that up).